docs/pages/documentation/data/table/api.js

```diff
@@ -129,6 +129,20 @@ export default [
                 default: '<code>false</code>'
             },
             {
+                name: '<code>detailedRows</code>',
```

Comment:
I think a better name for this is: "openedDetailed"
should be snake_case for API and example